### ISSUE TITLE
After adding new network and switching to it metamask shows additional message which prevents further interactions

### ISSUE THREAD

**Issue by domalaq** [Synthetixio/synpress#761]
So I call a method `cy.allowMetamaskToAddAndSwitchNetwork();` which does its job well and network is added and switched. But Metamask shows a modal message with title "You have switched to BNB (for example). Things to keep in mind ..." and button "Got it". And this message is not dismissed. And further operation like `cy.disconnectMetamaskWalletFromDapp();` can not be successfully executed.

**Expected behavior**
Modal message should be closed.

**Screenshots**
![image](https://github.com/Synthetixio/synpress/assets/3232885/ef6b1fc6-73ff-4e76-892f-b90dbf5b4ce7)

**Desktop (please complete the following information):**
 - OS: Pop_OS! 22.04
 - Synpress version: 3.7.0
- CI platform (if applicable): local testing

**Comment by domalaq** [comment]
Also if someone can suggest a workaround would be much appreciated.

**Comment by juantor16** [comment]
Hey @domalaq please share the way you are calling the cy.allowMetamaskToAddAndSwitchNetwork() method so I can reproduce it and see if can share a solution.

**Comment by juantor16** [comment]
> Hey @domalaq please share the way you are calling the cy.allowMetamaskToAddAndSwitchNetwork() method so I can reproduce it and see if can share a solution.

I got it, was able to reproduce, looking at a possible solution.

**Comment by juantor16** [comment]
@domalaq I have pushed a possible solution to the issue:

https://github.com/Synthetixio/synpress/pull/763

synthetixio/synpress#55237a0c06895bf996db90723aa429213ad3fcfe

**Comment by domalaq** [comment]
Hey thanks man. I'll check in a day or two, just letting you know.

**Comment by domalaq** [comment]
heya @juantor16 

Your fix works! I'll use it until it gets accepted.

Thank you very much!

**Comment by domalaq** [comment]
Ok I found that usual `allowNetworkToSwitch()` is broken now. It tries to find additional popup but fails. I think there should be some condition like _if there is additional popup then close otherwise proceed_.

**Comment by juantor16** [comment]
> Ok I found that usual `allowNetworkToSwitch()` is broken now. It tries to find additional popup but fails. I think there should be some condition like _if there is additional popup then close otherwise proceed_.

yeah makes sense, will push the new version today.

**Comment by juantor16** [comment]
Try this @domalaq d0c291b5ec2f11360b46e0da237725412d5f6df4

**Comment by domalaq** [comment]
Your new changes didn't work. I looked at your code and tried this snippet locally which worked:
```
    if ((await playwright
        .metamaskWindow()
        .locator(recipientPopupElements.popupCloseButton)
        .count()) > 0) {
      await playwright.waitAndClick(
        recipientPopupElements.popupCloseButton,
      )
    }
```

Also I think GotItButton is not needed to be declared. You don't use it anyway.
Looking forward to your working code! Thanks for contributing on this.

**Comment by duckception** [comment]
Hey @domalaq @juantor16, the version with the fix for this is available under `@synthetixio/synpress@3.7.2-beta.4`!
